source/Nuke.Common/Execution/Telemetry.cs

```diff
@@ -61,7 +61,10 @@ string GetCookieFile(string name, int version)
                 var cookieFile = GetCookieFile(cookieName, CurrentVersion);
                 if (!File.Exists(cookieFile))
                 {
-                    PrintDisclosure($"create awareness cookie for {cookieName.SingleQuote()}");
```

Comment:
This was crashing - it was prompting for `Console.ReadKey`. Not sure how this code path was working on build servers :confused:

Reply:
fixed in 6.0.2

Reply:
Fixed here - https://github.com/nuke-build/nuke/commit/c748538e1485aaaa7788639fd9d7887cba2f31c3#diff-88b57f7242c225bafbc30a6ad521e8204ad5a0e55e3c602cfefde33f93ba812bR109-R121